cypress/fixtures/console/devices/device.ns.json

```diff
@@ -0,0 +1,49 @@
+{
+  "end_device":{
+    "frequency_plan_id":"EU_863_870_TTN",
+    "lorawan_phy_version":"PHY_V1_0_2_REV_B",
+    "multicast":false,
+    "supports_join":false,
```

Comment:
ABP devices are quite rare in the real world, so I think we should not use them by default in the device mock. Two things could be done here:

1.  Create a separate mock for ABP devices
2.  Create a way to pass overwrites per component to the mock device within `createMockDeviceAllComponents`, eg. `createMockDeviceAllComponents({ ns: { end_device: { supports_join: false }}})`

Then merging those like

```js
  cy.fixture('console/devices/device.ns.json').then(body => {
    cy.request({
      method: 'PUT',
      url: `${baseUrl}/api/v3/ns/applications/${applicationId}/devices/${body.end_device.ids.device_id}`,
      { ...body, ...overwrites.ns },
      headers,
    })
  })
```

I think I'd prefer 2. since it is more flexible and requires fewer fixtures.